### PR TITLE
Kruize-fizzbuzz++/java/zach

### PR DESCRIPTION
**Pull request by Tech-Wizz**
Modified the code Zach provided to produce FizzBuzz with an added Ping 7 and Pong 11. Tested it with Zach's instructions and it worked correctly. If needed I can modify the instructions.

### CODE REVIEW

**Review by Skeegan123 — COMMENTED** [review]
Im not sure that it is a requirement, but this can definitley be simplified. You could use only 4 or 5 ifs to check if the number is divisible by 3, 5, 7, or 11. Then you can concat whatever word to the output string. Again I dont know if it is a requirement or not but probably wouldnt be a bad idea. Otherwise looks good. Ill give the +1 now and let me know if you want to make that change or not then Ill pull it down and test it.
EDIT: I realized that I forgot to look for tests so I am gonna take back that +1 for now lol.

File: fizzbuzz/java/Zach/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            }else if(i % 3 == 0) {
+                System.out.println("Fizz");
+            }else if (i % 5 == 0) {
+                System.out.println("Buzz");
+            }else if(i % 7 == 0){
+                System.out.println("Ping");
+            }else if(i % 11 == 0){
+                System.out.println("Pong");
+            }else {
+                System.out.println(i);
```
Comment (sleddog) [inline fizzbuzz/java/Zach/README.md:111]: there is a much easier way of doing this.  try to only do one comparison for each number, as opposed to enumerating all of these permutations

File: fizzbuzz/python/Andrew/fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            print("Fizz")
+        elif i % 5 == 0:
+            print("Buzz")
+        else:
+            print(i)
+
+if __name__ == "__main__":
+    #Get the integer input from the command-line
+    number = int(sys.argv[1])
+    fizz_buzz(number)
```
Comment (sleddog) [inline fizzbuzz/python/Andrew/fizzbuzz.py:19]: what is this file doing in here?  did you merge main back in?

File: fizzbuzz/java/Zach/README.md (hunk 1 above)
Comment (Tech-Wizz) [reply fizzbuzz/java/Zach/README.md]: I made it more simplified

File: fizzbuzz/python/Andrew/fizzbuzz.py (hunk 2 above)
Comment (Tech-Wizz) [reply fizzbuzz/python/Andrew/fizzbuzz.py]: I fixed it but I don't know why that was there.

**Comment by Tech-Wizz** [comment]
> Im not sure that it is a requirement, but this can definitley be simplified. You could use only 4 or 5 ifs to check if the number is divisible by 3, 5, 7, or 11. Then you can concat whatever word to the output string. Again I dont know if it is a requirement or not but probably wouldnt be a bad idea. Otherwise looks good. Ill give the +1 now and let me know if you want to make that change or not then Ill pull it down and test it. EDIT: I realized that I forgot to look for tests so I am gonna take back that +1 for now lol. EDIT 2: I see what you were trying to do for unit tests now but I dont think that is what Devin is looking for. You will probably want to use a library like JUnit to make the tests for your program. From there you can test the function. I am not familiar with JUnit so you will need to look up exactly how to do that yourself.

I found a much more simplistic way of testing. I tried JUnit but I couldn't figure out how to make it work in the terminal. It really did not like it.

**Review by Skeegan123 — COMMENTED** [review]
Alright this is a lot better but I still have a couple notes. For 1 you dont need this many tests. We only need to test 3, 5, 7, and 11 on their own. I dont think it hurts necessarily but it isnt needed. Also, how you specifically did it, you would need the line 
```
if ((i % 3 != 0) && (i % 5 != 0) && (i % 7 != 0) && (i % 11 != 0)) {
    System.out.print(i);
}
```
But you could get rid of this long if statement if you instead use a string and only output it at the end. The only reason I mention this is because Devin made a comment on Cameron's submission about a very similar line and how it should be simplified. Final question I have is, did you delete Andrews files from your local repository or git rm them? Because it looks like you deleted them and I am not sure that is the intended outcome. Let me know.
Edit: After looking back at Cameron's submission, he was already using a string to add the words too. Since you arent I dont know that you are expected to simplify that last if statement. Might be something to ask Devin but up to you

**Comment by Tech-Wizz** [comment]
I used git rm to keep them out of my repository. I am not sure why they where in my branch or trying to update in my branch.

**Review by Skeegan123 — COMMENTED** [review]
Got it. Well in that case it looks pretty good. Ill pull it down rn and test it. +1

**Review by Skeegan123 — APPROVED** [review]
Alright just ran it and it looks good! Im still having problems where there arent the correct permissions but I am starting to think it might just be my computer. So maybe try one more time using the command chmod +x run_tests.sh and commiting that file again but if not then I think its just my computer. Good Job!

**Pull request merged**